src/frontend/src/lib/components/tokens/TokenBalance.svelte

```diff
@@ -17,10 +17,7 @@
 				unitName: data.decimals
 			})}
 		{:else}
-			{formatToken({
-				value: ZERO,
-				unitName: data.decimals
-			}).replace('0', '-')}
+			<span>{$i18n.tokens.balance.error.not_applicable}</span>
```

Comment:
here will be the same, but there are less tokens where this is true; maybe here just greyed out?

Reply:
ditto